media/com_languages/js/admin-language-change-flag.es6.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+document.addEventListener('DOMContentLoaded', () => {
+  [].forEach.call(document.querySelectorAll('#jform_image'), (el) => {
+    el.addEventListener('change', (event) => {
+      const flag = event.currentTarget.value;
+      const flagimage = document.querySelector('#flag img');
```

Comment:
Please can we use:

```
const flag = document.getElementById('#flag');
const flag = flag.querySelector('#img');
```

Performance test to confirm: https://jsperf.com/justatestforme